test/controllers/error.controller.unit.ts

```diff
@@ -0,0 +1,17 @@
+jest.mock("ioredis");
+
+import mocks from "../mocks/all.middleware.mock";
+import request from "supertest";
+import app from "../../src/app";
+
+const EXPECTED_TEXT = "Page not found - File a confirmation statement";
+const INCORRECT_URL = "/confirmation-statement/company-numberr";
+
+describe("error controller test", () => {
+  it("should return page not found screen if page url is not recognised", async () => {
+    const response = await request(app)
+      .get(INCORRECT_URL);
+    expect(response.text).toContain(EXPECTED_TEXT);
```

Comment:
You should also expect the response.status to equal 404.
I think you'll need to modify the error controller to actually do this